AdaptorsDataAccessPlugins/src/au/gov/asd/tac/constellations/dataaccess/adaptors/DataAccessPluginAdaptorsType.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2010-2020 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellations.dataaccess.adaptors;
+
+import au.gov.asd.tac.constellation.views.dataaccess.DataAccessPluginType;
+import java.util.ArrayList;
+import java.util.List;
+import org.openide.util.lookup.ServiceProvider;
+
+/**
+ *
+ * @author GCHQDeveloper601
+ */
+@ServiceProvider(service = DataAccessPluginType.class)
+public class DataAccessPluginAdaptorsType extends DataAccessPluginType {
+
+    public static final String GAFFER_ROAD_EXAMPLE = "Gaffer Road Example";
```

Comment:
Plugin types are supposed to be used to broadly categorise plugins in the Data Access View. I'm not sure that a specific example plugin deserves its own category - i'd suggest using `DataAccessPluginCoreType.EXPERIMENTAL` or `DataAccessPluginCoreType.DEVELOPER` instead or perhaps creating something like `DataAccessPluginAdaptorType.EXAMPLE`.

Reply:
I went with DataAccessPluginAdaptorType.EXAMPLE